crates/ln-dlc-node/src/ln/event_handler.rs

```diff
@@ -364,21 +356,8 @@ where
                     forwarding_channel_manager.process_pending_htlc_forwards();
                 });
             }
-            Event::SpendableOutputs { outputs } => {
-                tracing::debug!(?outputs, "Spendable outputs");
-                let destination_address = self.wallet.inner().get_unused_address()?;
-                let output_descriptors = &outputs.iter().collect::<Vec<_>>();
-                let tx_feerate = self
-                    .wallet
-                    .get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
-                let spending_tx = self.keys_manager.spend_spendable_outputs(
-                    output_descriptors,
-                    Vec::new(),
-                    destination_address.script_pubkey(),
-                    tx_feerate,
-                    &Secp256k1::new(),
-                )?;
-                self.wallet.broadcast_transaction(&spending_tx);
+            Event::SpendableOutputs { .. } => {
+                // Spendable outputs are sent to the node's on-chain wallet directly
```

Comment:
You could log here that this should never happen :)

Reply:
Not quite, LDK still generates the event regardless of how we configure the channel or how we generate the final addresses.

Reply:
oh right. Thanks for clarification